src/main/java/com/gomulka/ola/dao/GeneratorExcelDao.java

```diff
@@ -12,26 +14,25 @@
 import java.util.Set;
 
 
-public class GeneratorExcelFile {
-    private XSSFWorkbook workbook;
-    private List<XSSFSheet> yearSheet;
-    private XSSFSheet sheet;
-    private ListService listService;
-    private List<GasData> gasList;
-    private GasService gs;
-    private Object[][] result;
-    private Set<String> yearSet;
-    private List<Object[][]> yearList;
-    private Object[][] titles = {{"date", "gas value", "temperature"}};
-
-    public GeneratorExcelFile(GasService gs) {
-        this.gs = gs;
-        this.listService = gs.getListService();
-        this.gasList = gs.getListService().getList();
-        this.yearSet = listService.getYearSet();
-
+public class GeneratorExcelDao {
```

Comment:
to juz nie powinno byc dao
DAO data access object
ExcelGenerator powinno byc ok